src/Microsoft.Azure.Functions.ExtensionBundle/bundleConfig.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
     "bundleId":  "Microsoft.Azure.Functions.ExtensionBundle.Preview",
-    "bundleVersion":  "4.26.0",
+    "bundleVersion":  "4.26.2",

```

Comment:
we should just take this change from hotfix and leave others as we don't want to pin the versions anymore